imagePicker.py

```diff
@@ -113,9 +115,9 @@ def main():
 
 
 def GetName(AutoOrCard):
-    if str(type(AutoOrCard)) == "<class 'scrython.cards.named.Named'>":
+    if isinstance(AutoOrCard, Named):
         return AutoOrCard.name()
-    if str(type(AutoOrCard)) == "<class 'scrython.cards.autocomplete.Autocomplete'>":

```

Comment:
OK, this makes sense now, and is also why I was having trouble getting this to work.  I didn't have the correct imports.  Thank you!